src/screen/CameraScreen.jsx

```diff
@@ -22,6 +15,8 @@ function CameraScreen({ navigation }) {
     Camera.Constants.FlashMode.off,
   );
   const cameraRef = useRef(null);
+  const params = route?.params;
+  const videoUrl = params?.uri;
```

Comment:
videoUrl이 기존 비디오를 뜻하니까 originVideo 가 어떨까요?
```suggestion
  const originVideo = params?.uri;
```

Reply:
넵 좋은 거 같습니다! 수정하겠습니다!